Updated "Microsoft Edge Add-ons API Reference" endpoints (#1695)
MicrosoftDocs/edge-developer: microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

> 

Reviewer: In section "Check the status of a package upload", cross-link back to the Using article's equivalent section.
Repeat for other sections.

Author: All 4 APis are already referenced, Can you please clarify what is the ask here?

Reviewer: I plan to add hyperlinks from h2 sections in the Reference article linking to the equivalent sections in the Using article, if they don't exist yet.  I plan to push a commit into this PR.

Reviewer: Done, in the sibling PR instead.  Cross-linked as See Also h3 sections.  That is, restored my original design from v1 of these articles.

| Method | Request URI |
|---|---|

> | `POST` | `/products/{productID}/submissions/draft/package` |

Reviewer: Globally check in both files, whether the variable is named "productID" or "ProductID".

Author: It is productID

#### Response headers

> *  Location: `{operationID}`

Reviewer: ```suggestion
*  Location: `{OperationID}`
```
Global: lowercase o or uppercase O

Author: updated to lowercase o

#### Response headers

> * Location: `{operationID}`

Reviewer: ```suggestion
* Location: `{OperationID}`
```
Global: lowercase o or uppercase O

```json
{
    "id": "{operationID}",
    "createdTime": "Date Time",
    "lastUpdatedTime": " Date Time ",
    "status": "Failed",
    "message": "Can't publish extension as your extension submission is in progress. Please try again later.",
    "errorCode": "InProgressSubmission",
    "errors": null    
}
```

#### Response when there is an ongoing unpublished submission for the same product

```json
{
    "id": "{operationID}",
    "createdTime": "Date Time",
    "lastUpdatedTime": " Date Time ",
    "status": "Failed",
    "message": "Can't publish extension as your extension is being unpublished. Please try after you've unpublished.",
    "errorCode": "UnpublishInProgress",
    "errors": null    
}
```

#### Response where any of the modules are invalid

```json
{
    "id": "{operationID}",
    "createdTime": "Date Time",
    "lastUpdatedTime": " Date Time ",
    "status": "Failed",
    "message": "Can't publish extension as your extension has modules that are not valid. Fix the modules with errors and try to publish again.",
    "errorCode": "ModuleStateUnPublishable",
    "errors": [
        {
            "message": "Invalid module : <Modules>"
        }
    ]
}
```

> #### Response when there are validation error(s) in submission

Reviewer: ```suggestion
#### Response when there are validation errors in submission
```
